### ISSUE TITLE
Formatting Issue in sample.env

### ISSUE THREAD

**Issue by iamirrf** [RunestoneInteractive/rs#411]
**Reported by:** @iamirrf

**Description:**

The `sample.env` file contains unnecessary spaces around the equals sign (`=`) in its key-value pairs, which can disrupt the setup process when configuring the `bash.rc` file. Proper configuration files should not have spaces around the equals sign. 

**Examples:*

- Incorrect: `REDIS_HOST = redis`
- Correct: `REDIS_HOST=redis`

Additionally, the default path in the `sample.env` file for `BOOK_PATH` seems outdated, suggesting an older project setup. 

- Current: `BOOK_PATH=~/Runestone/books`
- Suggested: `BOOK_PATH=~/books`

**Comment by bnmnetp** [comment]
These are not meant to be bashrc files. These are meant to be .env files.

**Comment by pearcej** [comment]
True, @bnmnetp, but the Runestone folder likely doesn't exist with a directly forked and clone rs repo, somthe key change is int he Additionally section. Furthermore, for those running in production mode, the lack of those spaces is likely helpful, and it changes nothing in development mode.

**Comment by bnmnetp** [comment]
The `BOOK_PATH` can be anywhere on the filesystem.  There is not a right answer to this.  For all I know someone could easily create a Runestone folder and clone the `rs` repository there.  Most people do not clone things into their home directory, most have a `projects` or `src` or something else kind of folder for the things they are working on.

I'm trying to give examples, not be prescriptive.
